doc/decisions/5_implemented/decision_process.md

```diff
@@ -73,21 +75,20 @@ Decisions by supervisors are undemocratic and decisions in meetings are nontrans
   Nevertheless they don't want to avoid other solutions, are open to arguments/facts/etc. and incorporate all input of decision PR fairly.
 - We will always be able to reach an consensus even if it requires that the core or plugins get multiple implementations.
   We don't need a vote (besides the approved review) or a benevolent dictatorship.
```

Comment:
Do you really think that there will always be a consensus?
Is having multiple implementations in the core or in plugins really a good idea? What if future decisions lead to adoptions that wouldn't be possible in all implementations? Wouldn't that add technical debt in the long run?

Reply:
> Do you really think that there will always be a consensus?

I updated:

```
+- For important decisions, we will able to reach an consensus even if it requires that the core or plugins get multiple implementations.
+  We accept indefinite postponement of lesser important decisions, where there is strong disagreement about the problem or the best solution.
+  Thus we don't need a vote (besides the approved review) or a benevolent dictatorship.
```

0a7d274b9209d12b37856dca40067c238f3ad253

> Is having multiple implementations in the core or in plugins really a good idea?

If there are different requirements/goals: Why not?

> What if future decisions lead to adoptions that wouldn't be possible in all implementations?

I don't know, do you have an example of what this could be?